testgen/generators.go

```diff
@@ -989,7 +989,7 @@ var EthSendRawTransaction = MethodTests{
 					GasPrice: new(big.Int).Add(genesis.BaseFee(), big.NewInt(1)),
 					Data:     common.FromHex("5544"),
 				}
-				s := types.MakeSigner(t.chain.Config(), t.chain.CurrentHeader().Number)
+				s := types.MakeSigner(t.chain.Config(), t.chain.CurrentHeader().Number, t.chain.CurrentBlock().Time)
```

Comment:
```suggestion
				s := types.LatestSigner(t.chain.Config())
```
Can you update all of these please? This is a little simpler.

Reply:
Hi @lightclient , changes are done :)